python-sdk/src/astro/constants.py

```diff
@@ -39,6 +39,7 @@ class FileType(Enum):
     JSON = "json"
     NDJSON = "ndjson"
     PARQUET = "parquet"
+    EXCEL = "xls,xlsx"
```

Comment:
Since Excel implements [many different formats](https://support.microsoft.com/en-au/office/file-formats-that-are-supported-in-excel-0943ff2c-6014-4e8d-aaea-b83d51d46247), apparently 14 MS-specific file formats to date, it is better to name the file format by the extension, e.g.:

```suggestion
    XLS = "xls"
    XLSX = "xlsx"
```

Reply:
Thanks, but in the case of 2 or more types it would require separate FileType's descendants for each of them, like XlsFileType and XlsxFileType. Currently you could just have all of them in one line - as you might have noticed, get_filetype() now can handle multiple extensions per type.
But I don't mind, if you wish, I'll do it.

Reply:
Why would we need different classes (`XlsFileType` and `XlsFileType`)..?

In `types/__init__.py`, we could have two entries pointing to the same class, for instance:
```
filetype_to_class: dict[FileTypeConstants, type[FileType]] = {
        FileTypeConstants.XLS: ExcelFileType,
        FileTypeConstants.XLSX: ExcelFileType
}
```
This way, if we decide to support other Excel formats, let's say `xml` or any new format MS decides to come up with, the user interface wouldn't change.

Reply:
@skolchin, please, if you could address this, we can merge this valuable contribution!

There is no such thing as one EXCEL file type. There are 14 file types. By replacing the generic `EXCEL` with specific ones, we are being future-proof if MS introduces more file formats or if we decide to support other Excel file formats.

Reply:
ok, will redo it this way

Reply:
@tatiana done